src/components/OrdersWidget/index.tsx

```diff
@@ -139,90 +174,95 @@ const OrdersWidget: React.FC = () => {
     <OrdersWrapper>
       <div>
         <h2>Your orders</h2>
-        <CreateButtons>
-          <Link to="/trade">
+        <CreateButtons className={noOrders ? 'withoutOrders' : 'withOrders'}>
+          {noOrders && (
+            <p className="noOrdersInfo">
+              It appears you haven&apos;t place any order yet. <br /> Create one!
+            </p>
+          )}
+          <Link to="/trade" className="tradeBtn">
             <ButtonWithIcon>
               <FontAwesomeIcon icon={faExchangeAlt} /> Trade
             </ButtonWithIcon>
           </Link>
-          <div className="strategy">
-            <ButtonWithIcon className="danger">
-              <FontAwesomeIcon icon={faChartLine} /> Create new strategy
-            </ButtonWithIcon>
-            <a href="/">
-              <small>Learn more about strategies</small>
-            </a>
-          </div>
+          <ButtonWithIcon className="danger strategyBtn">
+            <FontAwesomeIcon icon={faChartLine} /> Create new strategy
+          </ButtonWithIcon>
+          <a href="/" className="strategyInfo">
+            <small>Learn more about strategies</small>
+          </a>
         </CreateButtons>
       </div>
-      <OrdersForm>
-        <div className="infoContainer">
-          <div>
-            You have <Highlight>3</Highlight> standing orders
-          </div>
-          <div className="warning">
-            <FontAwesomeIcon icon={faExclamationTriangle} />
-            <strong> Low balance</strong>
+      {!noOrders && (
+        <OrdersForm>
+          <div className="infoContainer">
+            <div>
+              You have <Highlight>{orders.length}</Highlight> standing orders
+            </div>
+            <div className="warning">
+              <FontAwesomeIcon icon={faExclamationTriangle} />
+              <strong> Low balance</strong>
+            </div>
           </div>
-        </div>
-        <form action="submit">
-          <div className="ordersContainer">
-            <div className="headerRow">
-              <div></div> {/* Empty div on purpose */}
-              <div className="checked">
-                <input type="checkbox" />
-              </div>
-              <div className="title">Order details</div>
-              <div className="title">
-                Unfilled <br /> amount
+          <form action="submit">
+            <div className="ordersContainer">
+              <div className="headerRow">
+                <div></div> {/* Empty div on purpose */}
```

Comment:
Ups, I know someone it's going to complain https://github.com/gnosis/dex-react/pull/284